build/pause/Makefile

```diff
@@ -56,6 +57,11 @@ endif
 # If you want to build AND push all containers, see the 'all-push' rule.
 all: all-container
 
+all-push: all-push-images push-manifest
+
+push-manifest: manifest-tool
+	manifest-tool push from-args --platforms linux/amd64,linux/arm,linux/arm64,linux/ppc64le,linux/s390x --template $(REGISTRY)/pause-ARCH:$(TAG) --target $(REGISTRY)/pause:$(TAG)
```

Comment:
can you construct the `--platforms` arg from `$(ALL_ARCH)` somehow?

Reply:
similarly, rather than using `pause` here, it'd be nice if this could reuse the `$(IMAGE)` variable somehow